### PR TITLE
Added support for Chinese

### PR DESCRIPTION
**Pull request by liwuxi**
I've added support for Chinese, but I'm not sure it's quite accurate for some of the technical terms.

### CODE REVIEW

File: src/i18n/zh-CN.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  install: '安装',
+
+  posterizeInputImage: '对图像进行色彩处理',
+  colorSVG: '彩色 SVG',
+  monochromeSVG: '单色 SVG',
+
+  colorChannels: '色彩通道',
+  imageSizeAndRotation: '输入尺寸和旋转角度',
+  imagePreprocessing: '预处理',
+  svgOptions: 'SVG 选项',
```
Comment (zhihaozhang) [inline src/i18n/zh-CN.js:69]: Why svg got translation but image does not get translation in imageSizeAndRotation and imagePreprocessing ?

File: src/i18n/zh-CN.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+  considerDPR: '考虑设备像素比',
+
+  tweak: '调整',
+  closeOptions: '关闭',
```
Comment (zhihaozhang) [inline src/i18n/zh-CN.js:74]: Options did not got translation here ?

File: src/i18n/zh-CN.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+  optimizingSVG: '压缩 SVG',
+  copiedSVG: '复制 SVG',
+  savedSVG: '保存 SVG',
+
+  readyToWorkOffline: '准备离线工作。',
+  svgSize: 'SVG 大小',
+  bytes: '字节',
+  zoom: 'Zoom',
```
Comment (zhihaozhang) [inline src/i18n/zh-CN.js:83]: Should zoom be translated into ’缩放‘?

File: src/i18n/zh-CN.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  considerDPR: '考虑设备像素比',
+
+  tweak: '调整',
+  closeOptions: '关闭',
+
+  optimizingSVG: '压缩 SVG',
+  copiedSVG: '复制 SVG',
+  savedSVG: '保存 SVG',
+
+  readyToWorkOffline: '准备离线工作。',
```
Comment (zhihaozhang) [inline src/i18n/zh-CN.js:80]: I don't think 。 is needed here. Maybe translate into '离线工作就绪' is better ?

File: src/i18n/zh-CN.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  monochromeSVG: '单色 SVG',
+
+  colorChannels: '色彩通道',
+  imageSizeAndRotation: '输入尺寸和旋转角度',
+  imagePreprocessing: '预处理',
+  svgOptions: 'SVG 选项',
+
+  considerDPR: '考虑设备像素比',
+
+  tweak: '调整',
```
Comment (zhihaozhang) [inline src/i18n/zh-CN.js:73]: Maybe '微调' is better ?

File: src/i18n/zh-CN.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+  opacity: '透明度',
+  saturate: '饱和度',
+  sepia: '老化',
+
+  scale: '缩放',
+  rotation: '旋转',
+  turdsize: '抑制噪点',
+  alphamax: '边角阈值',
+  minPathSegments: '最小路径长度',
+  strokeWidth: '笔触大小',
```
Comment (zhihaozhang) [inline src/i18n/zh-CN.js:40]: 笔触宽度 should be better ?

File: src/i18n/zh-CN.js [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+  alpha: 'Alpha',
+
+  brightness: '亮度',
+  contrast: '对比度',
+  grayscale: '灰度',
+  'hue-rotate': '色调反转',
+  invert: '反转',
+  opacity: '透明度',
+  saturate: '饱和度',
+  sepia: '老化',
```
Comment (zhihaozhang) [inline src/i18n/zh-CN.js:33]: I think '褐度' should be better here.

File: src/i18n/zh-CN.js [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+
+  scale: '缩放',
+  rotation: '旋转',
+  turdsize: '抑制噪点',
+  alphamax: '边角阈值',
```
Comment (zhihaozhang) [inline src/i18n/zh-CN.js:38]: max is different from threshold.

File: src/i18n/zh-CN.js (hunk 1 above)
Comment (liwuxi) [reply src/i18n/zh-CN.js:69]: Because I'm translating from `en-US.js`, but anyway, I think it's fine to add.

File: src/i18n/zh-CN.js (hunk 2 above)
Comment (liwuxi) [reply src/i18n/zh-CN.js:74]: Same reason.

File: src/i18n/zh-CN.js (hunk 5 above)
Comment (liwuxi) [reply src/i18n/zh-CN.js:73]: This is the button used to open the Drawer in mobile devices. There are many options for adjustment, so I don't know if '微调' is appropriate.

File: src/i18n/zh-CN.js (hunk 7 above)
Comment (liwuxi) [reply src/i18n/zh-CN.js:33]: LOL, this word makes me hesitate for a long time.

File: src/i18n/zh-CN.js (hunk 8 above)
Comment (liwuxi) [reply src/i18n/zh-CN.js:38]: Same reason.

File: src/i18n/zh-CN.js (hunk 6 above)
Comment (liwuxi) [reply src/i18n/zh-CN.js:40]: I think '大小' is more semantic, like other image processing apps, correct me if I'm wrong.

File: src/i18n/zh-CN.js (hunk 4 above)
Comment (liwuxi) [reply src/i18n/zh-CN.js:80]: The '。' was added for the same reason.

**Comment by liwuxi** [comment]
In response to @zhihaozhang's comments made part of the optimization

**Review by zhihaozhang — APPROVED** [review]
LGTM

**Comment by tomayac** [comment]
Thank you very much for adding Chinese!

**Pull request merged**